### ISSUE TITLE
script-indent broke after updating

### ISSUE THREAD

**Issue by donnysim** [vuejs/eslint-plugin-vue#407]
**Tell us about your environment**

* **ESLint Version:** 4.18.1
* **eslint-plugin-vue Version:** 4.2.2, 4.3.0
* **Node Version:** 8.7.0

**Please show your full configuration:**

```
module.exports = {
    parserOptions: {
        parser: 'babel-eslint',
        ecmaVersion: 6,
        sourceType: 'module',
        ecmaFeatures: {
            experimentalObjectRestSpread: true,
        },
    },
    env: {
        browser: true,
    },
    plugins: [
        'vue'
    ],
    extends: [
        'plugin:vue/base',
    ],
    rules: {

    },
    overrides: [
        {
            files: ['*.vue'],
            rules: {
                'indent': 'off',
                'vue/script-indent': ['error', 4, {
                    baseIndent: 1,
                    switchCase: 2,
                }],
            },
        },
    ],
};
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<script>
    function test() {
        const variable = 0;

        if (variable === 0) {
            console.info('wrong indentation reported');
        }
    }

    test();
</script>
```

**What did you expect to happen?**
No errors.

**What actually happened? Please include the actual, raw output from ESLint.**
On the `if` line it throws `Expected indentation of 26 spaces but found 8 spaces`. Everything worked fine until today after updating the package. After downgrading to 4.2.2 the problem still exists so it's a dependency problem?

**Comment by qkdreyer** [comment]
Using eslint v4.17.0 or v4.18.1

test.js
```js
export default {
  x() {
    const x = 1;
    return x;
  },
};
```
output :
```
SublimeLinter: eslint: test.js ['/usr/local/bin/eslint', '--format', 'compact', '--stdin', '--stdin-filename', '@'] 
SublimeLinter: eslint output:
/private/tmp/test.js: line 4, col 1, Error - Expected indentation of 15 spaces but found 4 spaces. (vue/script-indent)
```

**Comment by qkdreyer** [comment]
https://github.com/vuejs/eslint-plugin-vue/commit/c6d25af2f1c02f0f1171edbacca51751f84153ee has introduced this issue.

Rollbacking from https://github.com/vuejs/eslint-plugin-vue/blob/c6d25af2f1c02f0f1171edbacca51751f84153ee/lib/utils/indent-common.js to https://raw.githubusercontent.com/vuejs/eslint-plugin-vue/5eaffdf2e1b0c62eec44e176f5059be0d178f549/lib/utils/indent-common.js fixes the issue.

**Comment by qkdreyer** [comment]
@mysticatea
